src/modal/ModalDialog.js

```diff
@@ -0,0 +1,99 @@
+import React, { useRef } from "react";
+import ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import useBreakpoint from "@w11r/use-breakpoint";
+
+import CloseModalSrc from "../assets/icons/closeModal.svg";
+
+const BackgroundAside = styled.aside`
+  align-items: center;
+  background-color: ${(props) => props.theme.colors.asideBackground};
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: ${(props) => props.theme.zIndex.modal};
+`;
+
+const ModalDialogWrapper = styled.div`
+  align-self: center;
+  background-color: ${(props) => props.theme.colors.background};
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: ${(props) => props.height};
+  max-height: 90vh;
+  max-width: 90wh;
+  overflow-y: auto;
+  padding: ${(props) => props.padding}px;
+  position: relative;
+  width: ${(props) => props.width};
+`;
+
+const CloseButtonImg = styled.img`
+  cursor: pointer;
+  height: 20px;
+  position: absolute;
+  right: ${(props) => props.position}px;
+  top: ${(props) => props.position}px;
+`;
+
+const DEFAULT_PADDING = 64;
+
+export default function ModalDialog(props) {
+  const { open, closeModal, height, width, children } = props;
+  const smallScreen = useBreakpoint(false, ["mobile-", true]);
+  const PADDING = smallScreen ? DEFAULT_PADDING / 4 : DEFAULT_PADDING;
+  const WIDTH = smallScreen ? "90%" : width;
```

Comment:
Realizing that this is likely an overstep and could lead to some unexpected behavior on smaller screens because we are ignoring the passed in `width` property.  The implementation modals should instead control their own width.

Reply:
@macfarlandian what do you think? Is this a blocker in your opinion?

Reply:
i'm not worried about it for now

Reply:
For now, I just left a warning comment in the code such that anyone who digs in will understand what's going on.  Won't be an issue for Tuesday, but long-term we'll likely want to do something more robust.